hacep/src/main/java/org/kie/hacep/Config.java

```diff
@@ -55,6 +55,8 @@
     public static final String TEST = Boolean.FALSE.toString();
     public static final String UNDER_TEST = "undertest";
     public static final String MAX_SNAPSHOT_REQUEST_ATTEMPTS = "max.snapshot.request.attempts";
+    public static final String UPDATABLE_KJAR = "UPDATABLEKJAR";
```

Comment:
Can you please add more info about these variables. It seems to me that these variables are not described in README.

Reply:
I'll a dd a section for all env vars in this PR https://github.com/kiegroup/openshift-drools-hacep/pull/59